app-doc/heirloom-doctools/files/README.gentoo-default

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
I think this variant doesn't really need the README. Just the two with detailed instructions.